mitiq/shadows/post_processing.py

```diff
@@ -0,0 +1,141 @@
+from typing import Tuple, List, Union, Any
+
+import numpy as np
+
+PAULI_MAP = {"X": 0, "Y": 1, "Z": 2}
```

Comment:
Something like this could simplify some code below. WDYT?
```suggestion
PAULI_MAP = {"X": hadamard, "Y": hadamard @ phase_z, "Z": identity}
```